docs/user/lectures.rst

```diff
@@ -46,14 +46,22 @@ Instructors have three options for creating a lecture.
     If instructors decide to process units automatically by clicking on the |process-units-checkbox| checkbox, additional information will be displayed
     and they can click on |process-units-btn| to process the lecture PDF into multiple units.
 
-       |create-lecture-automatic-unit-processing|
+       |automatic-unit-processing|
 
     Once the units are processed, instructors can view how the units are split, beginning from a start page to an end page. A name is automatically generated for each lecture unit.
     Instructors can also customize the table entries by removing or adding new units. Additionally, they can enter keywords if they want particular slides
     to be removed. Slides containing one of the comma-separated keywords will be removed.
 
        |processed-lecture-units|
 
+    The following video shows how to split a lecture PDF into multiple units.
+
+    .. raw:: html
+
+        <iframe src="https://live.rbg.tum.de/w/artemisintro/39407?video_only=1&t=0" allowfullscreen="1" frameborder="0" width="600" height="350">
```

Comment:
At least for me this is just the same video as 39409 above and talks about referencing single slides. Could you check this please?

Reply:
Users can reference single slides if instructors have uploaded lecture attachment units. So this is intentional, after the units are uploaded, this video is showing how to reference single slide.

Reply:
now it should be fixed 👍